### PR TITLE
Vfb 112 reset password link points to localhost 3000

### PR DESCRIPTION
**Pull request by minalui**
## What's changed
- Replaced hardcoded route for redirect at login page to {baseUrl}/auth/callback

## Screenshots / Videos
| Before     | After      |
|------------|------------|
| paste_here | paste_here |

## Checklist
- [x] The ticket is up-to-date - Please document any deviations from the original approach if there is any.
- [x] I have documented the testing steps for QA 
- [x] Make sure you've verified it works via `npm run dev`
- [x] Make sure you've verified it works via `npm run build` and `npm run start`
- [x] Make sure you've fixed all linting problems with `npm run lint_fix`
  - Will also be automatically checked on pull request
- [x] Make sure you've tested via `npm run test`
  - Will also be automatically checked on pull request

This will need to be checked through the deployed website***

### CODE REVIEW

File: src/components/LoginPanel.tsx [hunk 1]
```diff
@@ -54,10 +54,12 @@ const LoginPanel: React.FC<{}> = () => {
     const theme = useTheme();
 
     const [loaded, setLoaded] = React.useState(false);
+    const [baseUrl, setBaseUrl] = useState<string | null>(null);
 
     useEffect(() => {
+        setBaseUrl(window.location.origin);
         setLoaded(true);
-    }, []);
```
Comment (rickaw) [inline src/components/LoginPanel.tsx:60]: is this intentional?

File: src/components/LoginPanel.tsx [hunk 2]
```diff
@@ -54,10 +54,12 @@ const LoginPanel: React.FC<{}> = () => {
     const theme = useTheme();
 
     const [loaded, setLoaded] = React.useState(false);
```
Comment (rickaw) [inline src/components/LoginPanel.tsx:56]: let's keep this consistent

```suggestion
    const [loaded, setLoaded] = useState(false);
```

**Review by rickaw — APPROVED** [review]
Thanks! A few comments but feel free to merge once you're happy

File: src/components/LoginPanel.tsx (hunk 1 above)
Comment (minalui) [reply src/components/LoginPanel.tsx:60]: I've put baseUrl in the dependency - not sure why it thinks I've removed the full line.

**Pull request merged**